colossalai/inference/tensor_parallel/engine.py

```diff
@@ -46,6 +59,7 @@ def __init__(self,
                  max_input_len: int,
                  max_output_len: int,
                  dtype: torch.dtype = torch.float16,
+                 gptq: bool = False,
```

Comment:
Shall we move the `gptq` option to shard config or an inference config, instead of setting as an arg of TPInferEngine?

Reply:
Move option gptq to shard config.